syncer/syncer.go

```diff
@@ -371,7 +371,8 @@ func (s *Syncer) synchronize(ctx context.Context) bool {
 			s.setATXSynced()
 			return true
 		}
-		if len(s.dataFetcher.GetPeers()) == 0 {
+		// check that we have any peers
+		if len(s.dataFetcher.SelectBest(1)) == 0 {
```

Comment:
It will do wasted work to actually find the best one. Would it make sense to add a `GetPeersCount()` method?

Reply:
added  Total